lib/node_modules/@stdlib/math/base/special/asinf/test/test.js

```diff
@@ -126,3 +127,25 @@ tape( 'the function returns `NaN` if provided `NaN`', function test( t ) {
 	t.equal( isnanf( v ), true, 'returns NaN' );
 	t.end();
 });
+
+tape( 'the function returns `NaN` if provided a value less than `-1`', function test( t ) {
+	var v;
+	var i;
+
+	for ( i = 0; i < 1e3; i++ ) {
+		v = -( randu() * 1.0e6 ) - ( 1.0 - EPS );
```

Comment:
```suggestion
		v = uniform( -1.0e6, -1.0-EPS );
```

Let's clean this up by using `@stdlib/random/base/uniform` here, rather than `randu()`. Same for below.

Reply:
Modified this, along with `test.native.js` too, @kgryte.